modules/compression/src/bundle.js

```diff
@@ -1,7 +1,31 @@
-/* global window, global */
-const moduleExports = require('./index');
-const _global = typeof window === 'undefined' ? global : window;
-// @ts-ignore
-_global.loaders = _global.loaders || {};
-// @ts-ignore
-module.exports = Object.assign(_global.loaders, moduleExports);
+import * as moduleExports from './index';
+
+import {createWorker} from '@loaders.gl/worker-utils';
+import ZlibDeflateTransform from './lib/zlib/zlib-deflate-transform';
+import ZlibInflateTransform from './lib/zlib/zlib-inflate-transform';
+
+import LZ4DeflateTransform from './lib/zlib/zlib-deflate-transform';
+import LZ4InflateTransform from './lib/zlib/zlib-inflate-transform';
+
+export * from './index';
```

Comment:
This is different from the previous export. Doesn't it expose all classes in the global scope instead of as part of `loaders`?

Reply:
Just tried it locally, it indeed pollutes the global scope:

![image](https://user-images.githubusercontent.com/2059298/109335030-52480f80-7816-11eb-9d0a-ce11785d2343.png)

I'd recommend you stick with `export default globalThis.loaders` as the current behavior does.

Reply:
Interesting - just by adding it as a script? Did you actually build a local version and import as script?

I am working towards making this a true EcmaScript Module (ESM) that can be dynamically imported so that is why having the exports is desired. 

Will comment out `export *` for now and iterate. Perhaps we need to versions, or perhaps not using UMD could help?

Reply:
I wonder where the current default export goes? I believe script imports only use the global `loaders` variable that is explicitly set.

Reply:
Code updated

Reply:
This is still weird:

![image](https://user-images.githubusercontent.com/2059298/109344750-0734f900-7824-11eb-82a3-cd604dca36ee.png)

Maybe `export const loaders = globalThis.loaders`?

Reply:
I see. Regardless of how we handle that, we are assigning loaders variable explicitly so `window.loaders` should still be defined.

Reply:
I removed the default export and replace with an `export {moduleExports as loaders}`

I assume that there is some possibility of this interacting with the bundle's internal patching of `globalThis.loaders` when importing multiple scripts, might be worth testing again.

Reply:
`export {moduleExports as loaders}` overwrites the existing global loaders object.

You can test this by adding a `test.html` in the dist folder:

```html
<html>
  <body></body>
  <script src="https://unpkg.com/@loaders.gl/core@beta/dist/dist.min.js"></script>
  <script src="./dist.min.js"></script>
  <script>
    console.log(loaders);
  </script>
</html>
```